src/test/java/wolox/training/controllers/BookControllerTest.java

```diff
@@ -0,0 +1,113 @@
+package wolox.training.controllers;
+
+import static java.util.Optional.ofNullable;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(BookController.class)
+public class BookControllerTest {
+    @Autowired
+    private MockMvc mvc;
+
+    private Book bookTest;
+    private List<Book> books;
+
+    @MockBean
+    private BookRepository mockBookRepository;
+
+    private ObjectMapper objectMapper;
+
+    @PostConstruct
+    private void objectMapper() {
+        objectMapper = new ObjectMapper();
+    }
```

Comment:
You can inject an `ObjectMapper` instance from context
```suggestion
    @Autowired
    private ObjectMapper objectMapper;
```